### PR TITLE
Fix incorrect data type usage in add_step

### PR DESCRIPTION
**Pull request by keithf4**
Fix Issue https://github.com/omniti-labs/pg_jobmon/issues/14

### CODE REVIEW

**Comment by xzilla** [comment]
@keithf4 I pushed a PR which fixes the travis tests, and forced that into your other PR for the META file update. I think I can force all those changes into this PR too if you want, but haven't done it yet in case you want to resolve the conflicts yourself. Once we have this PR merged, perhaps we should do a new release?

**Comment by keithf4** [comment]
@xzilla Fixed the conflicts so should be good to go. Tagging a new release would be good then too.

File: sql/functions/add_step.sql [hunk 1]
```diff
@@ -19,7 +19,7 @@ $$;
 /*
  *  Add Step
  */
-CREATE OR REPLACE FUNCTION add_step(p_job_id bigint, p_action text) RETURNS bigint
+CREATE FUNCTION add_step(p_job_id bigint, p_action text) RETURNS bigint
```
Comment (xzilla) [inline sql/functions/add_step.sql:22]: I think this should still be create or replace, no?

Comment (keithf4) [reply sql/functions/add_step.sql:22]: I've always had the base files for a clean install not have the OR REPLACE so you know there's something wrong if that happens. Would mean that the extension is trying to replace an object that already exists in the database.

The update file has the OR REPLACE

**Pull request merged**